src/components/layout/Footer.astro

```diff
@@ -1,25 +1,32 @@
 ---
 import Svg from "@jasikpark/astro-svg-loader"
+
+const navLinks = [
+  { href: "/faq", text: "FAQ" },
+  { href: "/api", text: "API" },
+  { href: "/privacy", text: "Privacy" },
+  { href: "/about", text: "About" },
+  { href: "https://twitter.com/StormgateWorld", text: "Twitter", newWindow: true, twitter: true },
```

Comment:
optional ‘target’ and ‘icon’ would work better than newWindow/twitter/rss

Reply:
Right now, there is no icon. So I want wondering if this `<i>` was even used?

Reply:
ah icons are probably broken since astro upgrade, it uses https://tabler.io/icons/icon/brand-twitter-filled